honeybee_energy/ventcool/opening.py

```diff
@@ -209,6 +289,10 @@ def to_dict(self):
         base['fraction_height_operable'] = self.fraction_height_operable
         base['discharge_coefficient'] = self.discharge_coefficient
         base['wind_cross_vent'] = self.wind_cross_vent
+        base['air_mass_flow_coefficient_closed'] = self.air_mass_flow_coefficient_closed
```

Comment:
Maybe you want to do a check to see if this is not 0 before writing all of the AFN stuff into the dict.

Reply:
Will revisit this once we figure out default assumptions from discussion above.

Reply:
As I said above, there needs to be some way for people to disconnect the object from the AFN if they don't want the opening to have crack flow through it. So supporting 0 is very useful.

I'll also give another reason why we can't use `None` in the schema:
The C# bindings for the schema have much stricter type enforcement than the Python bindings. So this means that having both `None` and a float value is not acceptable. If there really was some way to have these values autocalculated, we should use a union of float with the [honeybee-core Autocalculate AltNumber](https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/altnumber.py#L37-L43). But I have been reserving this `Autocalculate` object  just for cases where EnergyPlus has an autocalculate feature. So, again, I think 0 is the best default here.

Reply:
Got it, will switch to 0, and document our assumptions.